api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -186,12 +193,13 @@
      * is equivalent to infinity; The operator hence will be attempted to be recovered indefinitely unless this value
      * is set to anything else.
      */
-    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<Integer>(new Integer2String());
+    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<>(new Integer2String());
```

Comment:
I removed that from some so that the line doesn't wrap. This was incorrectly removed for this line. I will change it back. Will not change everything because that is not part of fixing style violations.